src/extension.c

```diff
@@ -74,11 +74,18 @@ ts_extension_check_version(const char *so_version)
 		 */
 		ereport(FATAL,
 				(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
-				 errmsg("extension \"%s\" version mismatch: shared library version %s; SQL version "
+				 errmsg("extension \"%s\" version mismatch: shared library version %s; client "
+						"version "
 						"%s",
```

Comment:
Join lines.

Reply:
This PR doesn't change these lines. Furthermore, it was result of running formatting before this PR. And I disagree with joining lines, since it makes more difficult to understand.